### PR TITLE
IoTProjectManagedTest and DeviceRegistryTest

### PR DESCRIPTION
**Pull request by famartinrh**

### CODE REVIEW

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.IoTProjectManagedTest

File: systemtests/src/test/java/io/enmasse/systemtest/bases/IoTTestBase.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+import io.enmasse.iot.model.v1.IoTProject;
+import io.enmasse.systemtest.CustomLogger;
+import io.enmasse.systemtest.ability.ITestBase;
+import io.enmasse.systemtest.apiclients.IoTConfigApiClient;
+import io.enmasse.systemtest.apiclients.IoTProjectApiClient;
+import io.enmasse.systemtest.timemeasuring.SystemtestsOperation;
+import io.enmasse.systemtest.timemeasuring.TimeMeasuringSystem;
+import io.enmasse.systemtest.utils.IoTUtils;
+
+public abstract class IoTTestBase extends TestBase implements ITestBase{
```
[the remark this answers is not in the document]
Comment (famartinrh) [reply systemtests/src/test/java/io/enmasse/systemtest/bases/IoTTestBase.java]: true, forgot to delete that

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.IoTProjectManagedTest testprofile=systemtests-shared-iot

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.IoTProjectManagedTest testprofile=systemtests-shared-iot

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.IoTProjectManagedTest testprofile=systemtests-shared-iot

**Comment by kornys** [comment]
@famartinrh just note testprfile does not work, you need to use profile= if you want to specify maven profile

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.IoTProjectManagedTest profile=systemtests-shared-iot

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.IoTProjectManagedTest profile=systemtests-shared-iot

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.IoTProjectManagedTest profile=systemtests-shared-iot

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.IoTProjectManagedTest profile=systemtests-shared-iot

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.IoTProjectManagedTest profile=systemtests-shared-iot

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.IoTProjectManagedTest profile=systemtests-shared-iot

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.IoTProjectManagedTest profile=systemtests-shared-iot

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.IoTProjectManagedTest profile=systemtests-shared-iot

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.IoTProjectManagedTest profile=systemtests-shared-iot

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.IoTProjectManagedTest profile=systemtests-shared-iot

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.IoTProjectManagedTest profile=systemtests-shared-iot debug=true

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.IoTProjectManagedTest profile=systemtests-shared-iot

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.IoTProjectManagedTest profile=systemtests-shared-iot

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.IoTProjectManagedTest,**.DeviceRegistryTest profile=systemtests-shared-iot

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.IoTProjectManagedTest,**.DeviceRegistryTest profile=systemtests-shared-iot

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=io.enmasse.systemtest.iot.** profile=systemtests-shared-iot

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.DeviceRegistryTest profile=systemtests-shared-iot

**Comment by kornys** [comment]
@enmasse-ci run tests profile=smoke-iot

**Comment by kornys** [comment]
@enmasse-ci run tests profile=smoke-iot

**Comment by kornys** [comment]
@enmasse-ci run tests profile=smoke-iot

**Comment by kornys** [comment]
@enmasse-ci run tests profile=smoke-iot

**Comment by kornys** [comment]
@enmasse-ci run tests profile=smoke-iot

**Comment by kornys** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by kornys** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by kornys** [comment]
@enmasse-ci run tests profile=shared-iot

**Pull request merged**